spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue408Test.java

```diff
@@ -25,19 +22,15 @@ class Issue408Test extends AbstractIntegrationTest {
     @Test
     @DisabledOnJre(JRE.JAVA_8)
     void testSingleClass() {
-        Throwable throwable = Assertions.assertThrows(AssertionError.class, () -> {
-            performAnalysis("../java11/module-info.class");
-        });
+        Throwable throwable = Assertions.assertThrows(AssertionError.class, () -> performAnalysis("../java11/module-info.class"));
         Assertions.assertEquals("Analysis failed with exception", throwable.getMessage());
         assertThat(throwable.getCause(), is(instanceOf(IOException.class)));
     }
 
     @Test
     @DisabledOnJre(JRE.JAVA_8)
     void testFewClasses() {
-        performAnalysis("../java11/module-info.class", "../java11/Issue408.class");
-        BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder().build();
-        assertThat(getBugCollection(), containsExactly(0, bugTypeMatcher));
+        Assertions.assertDoesNotThrow(() -> performAnalysis("../java11/module-info.class", "../java11/Issue408.class"));
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/408) is about the analysis throwing exception.